### ISSUE TITLE
[0.49.3] Nimbus Theme Interstellar Map Pop Out Menu Un-clickable

### ISSUE THREAD

**Issue by HoneySkull** [MegaMek/mekhq#3191]
### Environment
MekHQ Version 0.49.3
OS: Windows 10
Java 11

### Description
In MekHQ - Change the UI theme to "Nimbus" in the File->Themes->Nimbus menu. 
Navigate to the Interstellar Map tab.
The in-map blue pop-up menu is not painted properly.  It is a solid blue square that is not clickable.
Changing the them to other themes allows this button to function properly.

### Files
![IntersellarMap_Nimbus](https://user-images.githubusercontent.com/83041327/160261846-8811ae40-b37b-4558-bae0-f0321feb024c.png)

**Comment by HammerGS** [comment]
We are currently up to 49.7 have you test this in that version.

**Comment by HoneySkull** [comment]
Yes - I just pulled today's (March 26th, 2022) code and tested with 0.49.8-SNAPSHOT and confirmed the issue still exists with the Nimbus theme.

**Comment by HoneySkull** [comment]
The issue also exists in the stable 0.48.0 - so nothing new introduced with recent changes.

**Comment by kernhe** [comment]
Can approve this bug OS:linux too

**Comment by Windchild292** [comment]
It's because of extra space below the button when using the Nimbus theme.
![image](https://user-images.githubusercontent.com/39067288/160293501-ded4a528-1d05-432d-9f3e-1dda59dcf477.png)

**Comment by IllianiCBT** [comment]
Nimbus theme was removed in 2024 due to numerous issues. Closing.
